src/sales/invoice-create.md

```diff
@@ -4,90 +4,108 @@ title: Creating an Invoice
+Normally, orders are invoiced and shipped after payment is received. However, if the method of payment is a purchase order, the order can be invoiced and shipped before payment is received. You can generate an invoice with a packing slip, and also print shipping labels from your carrier account. A single order can be divided into partial shipments which are invoiced separately, if necessary.
 
-{:.bs-callout-info}
-In the [payment method configuration]({% link configuration/sales/payment-methods.md %}), setting the new order status to _Processing_ will also make the _Automatically Invoice All Items_ option available. Some credit card payment methods complete the invoicing step as part of the process when the Payment Action is set to _Authorize and Capture_. In such a case, the _Invoice_ button does not appear and the order is ready to ship.
-
-## Payment actions
+When the state of new orders is set to _Processing_ the option to _Automatically Invoice All Items_ becomes available in the configuration. Some credit card payment methods complete the invoicing step as part of the process when [Payment Action](#payment-actions) is set to _Authorize and Capture_. In such a case, the Invoice button does not appear, and the order is ready to ship.
 
-|Payment action |Description
-|--- |---
-|Capture Online |When the invoice is submitted, the system captures the payment from the third-party payment gateway. You have the ability to create a credit memo and void the invoice.
-|Capture Offline |When the invoice is submitted, the system does not capture the payment. It is assumed that the payment is captured directly through the gateway, and you no longer have the option to capture this payment through Magento. You have the ability to create a credit memo, but you do not have the option to void the invoice. (Even though the order used an online payment, the invoice is essentially an offline invoice.)
-|Not Capture |When the invoice is submitted, the system does not capture the payment. It is assumed that you will capture the payment through Magento at a later date. There is a _Capture_ button in the completed invoice. Before capturing, you are able to cancel the invoice. After capturing you are able to create a credit memo and void the invoice.
+You must generate an invoice for an order before you can print it. To view or print the PDF, first download and install a PDF reader such as [Adobe Acrobat Reader][1].
 
-{:.bs-callout-info}
-Do not select the _Not Capture_ option unless you are certain that you are going to capture the payment through Magento at a later date. You cannot create a credit memo until the payment has been captured using the Capture button.
+![]({% link images/images/sales-invoices.png %}){: .zoom}
+_Invoices_
 
 ## Invoice an order
 
-You must generate an invoice for an order before you can print it. To view or print the PDF, you must have a PDF reader. You can download [Adobe Reader][1] at no charge.
+To create an invoice for an order:
```

Comment:
We have been omitting the infinitive intro when it is basically a repeat of the section heading. It is cleaner.